devZoGok/Battleship
# Ft 143 minimap

Implemented minimaps. To test:
1) load a map
2) observe the eye icon, representing the camera's location
3) observe own units, represented by the corresponding color's dots with a brighter circle around them
4) observe resource deposits, represented by gray icons
